components/Title.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Title = styled.div`
+  font-size: 26;
+  font-weight: 600;
+  text-align: center;
+  color:blue;
+`
+export default () => (
+  <>
+    <Title>Music Albums</Title>
+  </>
```

Comment:
Fragments are used when you have siblings you need to return, they are unnecessary here